### PR TITLE
Ubuntu install notes

### PR DESCRIPTION
**Pull request by crankymax**
Installation notes for Ubuntu linux distro.

### CODE REVIEW

**Comment by wcarthur** [comment]
The install notes mention a `linux_v20.yml` conda file. Can you add that into the repo as well? I do need to consider how we should manage conda environment files, but for now it's enough that it is included in the repo.

**Comment by crankymax** [comment]
Yup sorry about that.
I’ll add it after I finish backing up my VM.

**Comment by crankymax** [comment]
Looks like I decreased coverage. What would you like me to do? How do I increase coverage for a specification file?

**Comment by wcarthur** [comment]
That's a pretty insignificant decrease!? Probably just because the number of SLOC has increased, but the number of tested lines hasn't changed?

File: linux_v20.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  - backports.ssl-match-hostname==3.4.0.2
+  - click==6.7
+  - colormap==1.0.1
+  - flask==0.12.2
+  - itsdangerous==0.24
+  - jinja2==2.9.6
+  - markupsafe==1.0
+  - redis==2.10.5
+  - werkzeug==0.12.2
+prefix: /home/u53337/miniconda2/envs/tcrm
```
Comment (wcarthur) [inline linux_v20.yml]: Can this last line be removed? I'm wary of anything that has a fixed path in configurations

Comment (crankymax) [inline linux_v20.yml]: From https://stackoverflow.com/questions/41274007/anaconda-export-environment-file it doesn't look used, so it's just a stupid thing to have hanging around.

**Comment by wcarthur** [comment]
Cool - looks good to go now!

**Comment by wcarthur** [comment]
@crankymax I'll leave the issue_25 branch open for you to continue working in?

**Comment by crankymax** [comment]
I think that's good, if it suits you.

**Pull request merged**